pitaya-sharp/NPitaya/src/PitayaCluster.RPC.cs

```diff
@@ -46,16 +51,30 @@ private static async Task HandleIncomingRpc(IntPtr cRpcPtr)
 
                 Marshal.FreeHGlobal(res.data);
                 Marshal.FreeHGlobal(resPtr);
+                if (success)
+                {
+                    MetricsReporters.ReportTimer(Metrics.Constants.Status.success.ToString(), route,
+                        "handler", "", sw);
```

Comment:
Isn't the correct type "rpc" instead of "handler"? Following the pitaya patterns

Reply:
If I saw it correctly, the ReportTimer containing "rpc" is already implemented, so I just needed to implement the other one